lib/pdk/validators/metadata/metadata_json_lint.rb

```diff
@@ -26,7 +26,7 @@ def self.spinner_text(targets = [])
       end
 
       def self.pattern
-        'metadata.json'
+        '**/metadata.json'
```

Comment:
[PDK-401](https://tickets.puppetlabs.com/browse/PDK-401) specifies _Only validate <module_root>/metadata.json_ so I think this pattern should be unchanged.